ansible/files/bootstrap/telemetry.sh

```diff
@@ -137,6 +145,28 @@ sudo yum makecache
 # Install packages
 sudo yum -y install git jq htop tmux libffi-devel aws-cli postgresql-devel zsh snappy-devel readline-devel emacs nethogs w3m
 
+# Install agents for metrics and logging
+if [ "$METRICS_PROVIDER" = "stackdriver" ] || [ "$LOGGING_PROVIDER" = "stackdriver" ]; then
+    # Install GCP credentials for stackdriver; see https://cloud.google.com/logging/docs/agent/authorization
+    sudo mkdir -p /etc/google/auth/
+    sudo aws s3 cp $TELEMETRY_CONF_BUCKET/credentials/gcp.json /etc/google/auth/application_default_credentials.json
+fi
+if [ "$LOGGING_PROVIDER" = "stackdriver" ] ; then
+    # https://cloud.google.com/logging/docs/agent/installation
+    sudo bash -c "$(curl -sLS https://dl.google.com/cloudagents/install-logging-agent.sh)"
```

Comment:
This likely requires additional setup and configuration, esp. if this runs in AWS.

Reply:
That may be the case? The stackdriver docs specifically talk about installing on AWS and it sounds like simply providing creds and installing the agent is enough to get a default set of logs shipping, including syslog. My plan is to deploy this to a personal staging bucket once we have creds figured out and start testing what gets to GCP to discover if additional configuration is needed.

Reply:
A manual step for linking the aws account is required.  A service account + credentials then need to be created and placed on host.  The agent(fluentd) is preconfigured to send syslog + other things, but spark logs on emr likely require addl configuration.  I have outlined some steps https://mana.mozilla.org/wiki/display/SVCOPS/Stackdriver+Logging+for+AWS but this is for ec2 instances running containers